packages/wxt/src/client/content-scripts/content-script-context.ts

```diff
@@ -35,7 +35,9 @@ import { createLocationWatcher } from './location-watcher';
  * ```
  */
 export class ContentScriptContext implements AbortController {
-  private static SCRIPT_STARTED_MESSAGE_TYPE = 'wxt:content-script-started';
+  private static SCRIPT_STARTED_MESSAGE_TYPE = getUniqueEventName(
+    'wxt:content-script-started',
+  );
```

Comment:
I thought I was already using this function for this event... but I was using it for `wxt:locationchange`, must have misremembered where it was used.